.eslintrc

```diff
@@ -3,7 +3,11 @@
 	"plugins": ["prettier"],
 	"rules": {
 		"react/react-in-jsx-scope": "off",
-		"prettier/prettier": "error",
+		"prettier/prettier": "error", [
+		{
+		"endOfLine: "auto"
+		}
+		],
```

Comment:
Just wanted to check if there was a reason for changing the `endOfLine` config.

Reply:
@koredefashokun same reason. it's the setting for the dev who is working with windows but there is no need to commit to the PR.  @sebfk please reset this file in your commit.